arrows/klv/klv_convert_vital.cxx

```diff
@@ -236,25 +261,13 @@ klv_0104_to_vital_metadata( klv_timeline const& klv_data, uint64_t timestamp,
     vital_data.add< kv::VITAL_META_MISSION_NUMBER >( ss.str() );
   }
 
-  // Parse the datetime string into a UNIX microsecond timestamp
-  auto const start_datetime =
-    klv_data.at( standard, KLV_0104_START_DATETIME, timestamp );
-  if( start_datetime.valid() )
-  {
-    auto const value = start_datetime.get< std::string >();
-    vital_data.add< kv::VITAL_META_START_TIMESTAMP >(
-        kv::std_0104_datetime_to_unix_timestamp( value ) );
-  }
-
-  // Parse the datetime string into a UNIX microsecond timestamp
-  auto const event_start_datetime =
-    klv_data.at( standard, KLV_0104_START_DATETIME, timestamp );
-  if( start_datetime.valid() )
-  {
-    auto const value = event_start_datetime.get< std::string >();
-    vital_data.add< kv::VITAL_META_EVENT_START_TIMESTAMP >(
-        kv::std_0104_datetime_to_unix_timestamp( value ) );
-  }
+  // Parse the datetime strings into UNIX microsecond timestamps
```

Comment:
This comment might be better as the function description. And maybe rename the function to `klv_0104_parse_datetime_to_unix` so the comment isn't needed here?

Reply:
I use these comments to break up this large function into "sections" to improve readability, sort of like the `//---` function separators. So I would want to keep a comment here, even if not strictly necessary for its own sake. But I can change the function name if you think it improves comprehension at function definition.

Reply:
yep, that's fine. This looks good to me now.